assets/styles.css

```diff
@@ -1,3 +1,767 @@
+  font-weight: 800;
+  letter-spacing: 0.05em;
+}
+
+.interactive-table-text-one {
+  margin-top: -650px;
+  z-index: 5;
+  display: flex;
+  flex-direction: row;
+}
+
+.interactive-table-text-two {
+  margin-top: 240px;
+  z-index: 5;
+  display: flex;
+  flex-direction: row;
+}
+
+.interactive-table img {
+  height: auto;
+  max-width: 90%;
+}
+
+.interactive-footer {
+  margin-top: 30px;
+  font-size: 12px;
+  font-size: 0.85rem;
+  font-weight: 500;
+  font-style: normal;
+  font-family: nyt-franklin, arial, helvetica, sans-serif;
+}
+
+.interactive-footer::before {
+  width: 60px;
+  content: "";
+  display: inline-block;
+  height: 2px;
+  margin: 0;
+  background-color: #000;
+}
+
+.newsletters-section {
+  display: flex;
+  width: 1200px;
+  margin: 0 auto;
+  flex-direction: row;
+  padding-bottom: 20px;
+}
+
+.suggestions-title {
+  border-top: 2px solid rgb(0, 0, 0);
+  width: 75%;
+  margin-right: 10px;
+  display: flex;
+  flex-direction: row;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.suggestion-text {
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-weight: 700;
+  font-size: 1.0625rem;
+  line-height: 1.3125rem;
+  width: 80%;
+}
+
+.dont-show-text {
+  text-align: right;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-weight: 500;
+  font-size: 0.875rem;
+  line-height: 1.0625rem;
+  color: rgb(102, 102, 102);
+  background: rgb(255, 255, 255);
+  cursor: pointer;
+}
+
+.popular-title {
+  border-top: 2px solid rgb(0, 0, 0);
+  width: 25%;
+  margin-left: 10px;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-weight: 700;
+  font-size: 1.0625rem;
+  line-height: 1.3125rem;
+  padding-top: 20px;
+  border-bottom: 1px solid rgb(226, 226, 226);
+}
+
+.suggestions {
+  /* width: 80%; */
+  display: flex;
+  flex-direction: row;
+}
+
+.suggestions-columns {
+  width: 30%;
+  margin-left: 10px;
+  margin-right: 10px;
+  border-radius: 3px;
+  cursor: pointer;
+  border: 1px solid rgb(68, 67, 67);
+  height: 250px;
+  justify-content: space-between;
+}
+
+.popular-items {
+  padding-bottom: 10px;
+  padding-top: 20px;
+  border-bottom: 1px solid rgb(226, 226, 226);
+  font-size: 1.1rem;
+  font-weight: 700;
+  line-height: 1.375rem;
+  letter-spacing: 0.9px;
+  color: rgb(18, 18, 18);
+  font-family: nyt-cheltenham, georgia, "times new roman", times, serif;
+}
+
+.suggestions-one-box {
+  display: flex;
+  flex-direction: column;
+  padding: 5px;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-size: 0.9375rem;
+  line-height: 1.125rem;
+  letter-spacing: 0.2px;
+}
+
+.sgtn-img {
+  float: left;
+}
+
+.plus-svg {
+  float: right;
+}
+
+.row-one {
+  margin-top: 20px;
+  padding-bottom: 20px;
+  width: 97%;
+  border-top: 2px solid rgb(0, 0, 0);
+}
+
+.css-tadcwa {
+  margin-top: 20px;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-weight: 700;
+  font-size: 1rem;
+  line-height: 1.25rem;
+}
+
+.css-y3qb10 {
+  margin: 0;
+}
+
```

Comment:
Group this selector and selector on line 1, both have the same style definition

Reply:
Done